### PR TITLE
Add app service plan component

### PR DESCRIPTION
**Pull request by Flanker32**
Add app service plan UI component

### Creation
image

### Drop down
image

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/AppServicePlanPanel.java [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+import java.awt.Window;
+import java.awt.event.ItemListener;
+import java.awt.event.WindowAdapter;
+import java.awt.event.WindowEvent;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class AppServicePlanPanel extends JPanel {
+    public static final String CREATE_APP_SERVICE_PLAN = "Create app service plan";
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/AppServicePlanPanel.java]: private

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/AppServicePlanPanel.java [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+    private Window window;
+    private Subscription subscription;
+
+    private String subscriptionId;
+    private OperatingSystem operatingSystem;
+    private AppServicePlanWrapper selectedAppServicePlan = null;
+    private List<AppServicePlanWrapper> appServicePlanWrapperList = new ArrayList<>();
+
+    public AppServicePlanPanel() {
+        cbAppServicePlan.setRenderer(new ListCellRendererWrapper() {
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/AppServicePlanPanel.java]: SimpleListCellRenderer?

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/AppServicePlanPanel.java (hunk 1 above)
Comment (Flanker32) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/AppServicePlanPanel.java]: Fixed

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/AppServicePlanPanel.java (hunk 2 above)
Comment (Flanker32) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/AppServicePlanPanel.java]: Changed

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/NewAppServicePlanDialog.java [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+    /**
+     * Load pricing tier from model.
+     */
+    public void onLoadPricingTier() {
+        try {
+            cbPricing.removeAllItems();
+            final List<PricingTier> pricingTiers = AzureMvpModel.getInstance().listPricingTier();
+            pricingTiers.stream().forEach(pricingTier -> cbPricing.addItem(pricingTier));
+        } catch (IllegalAccessException e) {
+            e.printStackTrace();
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/NewAppServicePlanDialog.java]: Log this error

**Pull request merged**